### DISCUSSION TITLE
Do we have support for multi-select in mui forms ?

### DISCUSSION THREAD

**Discussion by venkatesh1146** [manojadams/mui-forms#54] [Q&A] [answered]
Hi @manojadams 
Do we have support for multi select? I am not able to see it in the docs.

**Comment by manojadams** [comment]
Currently it is not supported yet.
We might take it up in the upcoming version but not yet planned.

For now, you can make use of custom components to make use of same.

**Comment by manojadams** [comment, marked as answer]
@venkatesh1146 @venkateshpullaganti 
Current documentation for multiselect field

- https://mui-forms-docs.vercel.app/fields/multi-select

Note: This site is currently work in progress and will be merged to mui-forms later (when ready).
